package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "chulie",
-  "version": "0.6.0",
+  "version": "0.6.1",
```

Comment:
This should have a corresponding change in `package-lock.json`:
https://github.com/mycase/chulie/blob/52bb08162ddaf7a494fac1b44cd528d2844c330e/package-lock.json#L3

If we use `npm version patch` to update the version, it'll automatically take care of updating the version number in both files as well as creating a commit and a corresponding tag.

But, I actually think we should drop this commit from the PR. The recommended practice is to perform the version bump on its own, separate from any other changes, in order to avoid conflicts where two different PRs both try to update to the same version.

Reply:
We used to do this for gem releases: only bump version on master branch before a release is cut.  Because we don't have automatic build flows, a problem with that is the sometimes people forget to bump version before cut release.  Given that most of our gems and npm packages are not heavily developed, I think that we can still take the compromise to add a version bump in the branch and avoid an extra PR.

Reply:
package-lock.json updated in 2d4eeea

Reply:
> a problem with that is the sometimes people forget to bump version before cut release

I don't think this is a problem. Publishing the release will fail if we haven't changed the version; it won't overwrite the old version because that version is immutable.

> Given that most of our gems and npm packages are not heavily developed, I think that we can still take the compromise to add a version bump in the branch and avoid an extra PR.

While this repository may not get many concurrent PRs, I think it's a bad idea to adopt a habit that only works in repositories that are less active. If the only other reason is to avoid an extra PR, we can avoid that by bumping directly on `master`. (That's what I did in 52bb08162ddaf7a494fac1b44cd528d2844c330e, and I'm happy to do it again after this is merged.) And that way, we can follow the convention of using `npm version <semver-type>` to bump the version since it handles everything, including creating a tag. (That's also the workflow we've using to bump the version in https://github.com/mycase/tuscany, so it would be consistent ther.)

Reply:
I prefer that the package is published by a bot not a human.  But anyway, I'll get what you want and let's get my other story unblocked.

Reply:
Yeah, it'll be nice to automate the release flow going forward. I know there are some GitHub Actions related to npm releases, so we could investigate those. For now I'll cut a manual release so we can start using the new version right away 👍